examples/inverse/plot_mixed_source_space_inverse.py

```diff
@@ -0,0 +1,135 @@
+# source space bounds using the inner skull surface
+
+# Setup a volume source space
+# set pos=7.0 for speed issue
+vol_src = setup_volume_source_space(subject, mri=fname_aseg,
+                                    pos=7.0,
+                                    bem=fname_model,
+                                    volume_label=labels_vol,
+                                    subjects_dir=subjects_dir)
+# Generate the mixed source space
+src += vol_src
+
+n = sum(src[i]['nuse'] for i in range(len(src)))
+print('the src space contains %d spaces and %d points' % (len(src), n))
+
+# Write the mixed source space
+write_source_spaces(fname_mixed_src, src)
+
+# Export source positions to nift file
+nii_fname = op.join(bem_dir, '%s-mixed-src.nii' % subject)
+src.export_volume(nii_fname, mri_resolution=True)
+
+plotting.plot_img(nii_fname)
+plt.show()
+
+# Compute the fwd matrix
+fwd = make_forward_solution(fname_evoked, fname_trans, src, fname_bem,
+                            fname_fwd,
+                            mindist=5.0,  # ignore sources<=5mm from innerskull
+                            meg=True, eeg=False,
+                            n_jobs=1,
+                            overwrite=True)
+
+leadfield = fwd['sol']['data']
+print("Leadfield size : %d sensors x %d dipoles" % leadfield.shape)
+
+src_fwd = fwd['src']
+n = sum(src_fwd[i]['nuse'] for i in range(len(src_fwd)))
+print('the fwd src space contains %d spaces and %d points' % (len(src_fwd), n))
+
+# Load data
+condition = 'Left Auditory'
+evoked = mne.read_evokeds(fname_evoked, condition=condition,
+                          baseline=(None, 0))
+noise_cov = mne.read_cov(fname_cov)
+
+# Compute inverse solution and for each epoch
+snr = 3.0           # use smaller SNR for raw data
+inv_method = 'MNE'  # sLORETA, MNE, dSPM
+parc = 'aparc'      # the parcellation to use, e.g., 'aparc' 'aparc.a2009s'
+
+lambda2 = 1.0 / snr ** 2
+
+# Compute inverse operator
+inverse_operator = make_inverse_operator(evoked.info, fwd, noise_cov,
+                                         loose=None, depth=None,
+                                         fixed=False)
+
+stcs = apply_inverse(evoked, inverse_operator, lambda2, inv_method,
+                     pick_ori=None)
+
+# Get labels for FreeSurfer 'aparc' cortical parcellation with 34 labels/hemi
+labels_parc = mne.read_labels_from_annot(subject, parc=parc,
+                                         subjects_dir=subjects_dir)
+
+# Average the source estimates within each label of the cortical parcellation
+# and each sub structure contained in the src space
+# If mode = 'mean_flip' this option is used only for the surface cortical label
+src = inverse_operator['src']
+
+label_ts = mne.extract_label_time_course([stcs], labels_parc, src,
+                                         mode='mean',
+                                         allow_empty=True,
+                                         return_generator=False)
+
+# TODO some plot
```

Comment:
Following the CircleCI Artifacts build I only see the old version:

https://3227-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/inverse/plot_mixed_source_space_inverse.html

If you think it looks good, please go ahead and push the change, and if possible look at the CircleCI build and post the link (there should be an "Artifacts" tab at the top of the build info if you log into CircleCI).

I will say that the `plt.` access is not as preferred as the method-based one, so it's better to do this instead (assuming I have the right units, etc.):
```
fig, axes = plt.subplots(1)
axes.plot(1e3 * stcs.times, label_ts[0][0, :], 'k', label='bankssts-lh')
axes.plot(1e3 * stcs.times, label_ts[0][71, :].T, 'r',
          label='Brain-stem')
axes.set(xlabel='Time (ms)', ylabel='MNE current (nAm)')
axes.legend()
mne.viz.tight_layout()
plt.show()
```